test/Microsoft.AspNet.Mvc.ViewFeatures.Test/ViewComponents/DefaultViewComponentDescriptorProviderTest.cs

```diff
@@ -81,7 +81,10 @@ protected override IEnumerable<TypeInfo> GetCandidateTypes()
                     GetAssemblyProvider()
                     .CandidateAssemblies
                     .SelectMany(a => a.DefinedTypes)
-                    .Select(t => t.GetTypeInfo());
+#if DNX451
+                    .Select(t => t.GetTypeInfo())
+#endif
```

Comment:
`DefinedTypes` returns `TypeInfo` in DNXCore50 and `Type` in DNX451. It's kinda weird they broke this API.